.devcontainer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# To fully customize the contents of this image, use the following Dockerfile instead:
+# https://github.com/microsoft/vscode-dev-containers/tree/v0.128.0/containers/typescript-node-10/.devcontainer/Dockerfile
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:12
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        apt \
+        curl \
+        file \
+        git \
+        ninja-build \
+        net-tools \
+        ninja-build \
```

Comment:
ninja-build is here twice

Reply:
Probably accidental copy paste... I'll remove it.